hw_1.1/hw_1.1/Program.cs

```diff
@@ -0,0 +1,68 @@
+﻿using System;
+
+class Program
+{
+    static void swap(int[] array, int firstIndex, int secondIndex)
```

Comment:
```suggestion
    static void Swap(int[] array, int firstIndex, int secondIndex)
```
Методы нужно называть с большой буквы.